### ISSUE TITLE
Undo/Redo does not work in Notebook Cells after saving the notebook twice or more.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#45389]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.89.0
- OS Version: Windows_NT x64 10.0.19045

Steps to Reproduce:

1. Open a ipython notebook. (e.g. test.ipynb)
2. In a python cell, write code.
3. Do undo (`ctrl+z`) and redo (`ctrl+y`), it will work.
4. Save the notebook twice or more (`ctrl+s`, `ctrl+s`) .
5. Try undo (`ctrl+z`) and redo (`ctrl+y`) again, it doesn't work.

I tried to do the undo/redo from `Edit` menu. Still the same. (It's my habit to save multiple times on any editor or application!) 

TLDR: Undo redo doesn't work after saving the notebook twice.
